grammar/rainerscript.h

```diff
@@ -26,6 +26,9 @@
 #include <regex.h>
 #include "typedefs.h"
 
+#ifdef _AIX
+#define var var_tt
```

Comment:
Hi, I believe it's a better choice to rename concerned structs and typedefs as this make all future contributions compatible with AIX by design (except for unavoidable GCC pragmas).